packages/components/src/components/timeline/timeline.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import classNames from 'classnames';
+import Text from '../text';
+
+type TOval = {
+    children: number;
+};
+
+const Oval = ({ children }: TOval) => {
+    return (
+        <div className='dc-timeline__oval'>
+            <Text color='colored-background' size='s' weight='bold' className='dc-timeline__number'>
+                {children}
+            </Text>
+        </div>
+    );
+};
+
+type TItem = React.HTMLAttributes<HTMLDivElement> & { item_title: string | React.ReactNode };
```

Comment:
`string` is a part of `ReactNode` types, do we need to define the `string` again?

Reply:
yeah, but there is no item_title value in this type, so I had to add it

Reply:
@niloo-fs thanks, will check it